.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    name: Node.js ${{ matrix.node_version }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node_version:
+          - 13
+          - 12
+          - 10
+        os:
+          - ubuntu-latest
+          - windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node_version }}
+      - run: npm install
+      - run: npm run test
+      - uses: coverallsapp/github-action@master
+        continue-on-error: true
+        with:
+          github-token: ${{ secrets.github_token }}
+          parallel: true
+
+  coverage:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+        - uses: coverallsapp/github-action@master
+          continue-on-error: true
+          with:
+              github-token: ${{ secrets.github_token }}
+              parallel-finished: true
```

Comment:
Why cannot this just be part of the `test` job? I don't see what it does differently from the code in the `test` job either.

Reply:
This job send a webhook to coveralls, need wait for all test job finished.

See https://github.com/sindresorhus/eslint-plugin-unicorn/pull/504#discussion_r373785339 and https://github.com/sindresorhus/eslint-plugin-unicorn/pull/504#issuecomment-581056148

Reply:
Why though? We only need coveralls to run in one of the test job, not all.

Reply:
I removed `parallel`, and add a condition. only run on node.js 12 on linux